index.html

```diff
@@ -0,0 +1,17 @@
+//file <!DOCTYPE html>
+<html>
+  <head>
+    <meta charset="utf-8">
+    <link rel="stylesheet" type="text/css" href="styles/style.css">
+
+    </head>
+    <title></title>
+  </head>
+  <body>
+    <link rel="stylesheet" type="text/css" href="scripts/script.css">
```

Comment:
In the `<body>` section you should be adding a link to your javascript file (which currently has nothing in it!) Javascript files have names ending in .js, for example, script.js. In addition, javascript files are linked in a different way from the way that CSS files are. Instead of a `<link>` tag, you will use a `<script>` tag. (A good resource for more information is here: [HTML <script> src Attribute](https://www.w3schools.com/tags/att_script_src.asp) )

So the line should be:

``` html
   <script type="text/javascript" src="scripts/script.js"></script>
```

Finally, because javascript is a little picky, you'll find fewer problems if you put this line of code just before closing your `<body>` tag, for example, just before line 16 in your code.